docs/schema-loader.md

```diff
@@ -78,7 +78,7 @@ Create/Delete schemas in the storage defined in the config file
       --ru=<ru>       Base resource unit (supported in DynamoDB, Cosmos DB)
 ```
 
-For Cosmos DB (Deprecated. Please use the command using a config file instead):
+For Cosmos DB for NoSQL (Deprecated. Please use the command using a config file instead):
 ```console
 Usage: java -jar scalardb-schema-loader-<version>.jar --cosmos [-D]
```

Comment:
I don't update this part since these descriptions are defined in the source code of Schema Loader.